src/Features/App/App.tsx

```diff
@@ -108,10 +117,12 @@ export default function App() {
       </Suspense>
     );
   }
-      
+
   if (hasData) {
+    const canEditVerifiedTasks =
+      settingsQuery.data[0].config.find((setting: { key: string }) => setting.key === "enable.tasks").value ?? false;
```

Comment:
Agree with this point. There is currently only one element in the array for settings. But it would be safer to find the element in the array where `name === 'Workers'`.

 Also agree with your second point for this particular new flag. I'm leaning towards `editVerfifiedTasksEnabled`. "Enabled" feels like a good suffix for a flag variable. Perhaps "standaloneEnabled" or "standaloneModeEnabled"?

Reply:
@BenjaminRuby sorry just getting back to you here. I like prefixing booleans with an `is` but might not be necessary w/ TS as we know its type. Either one works for me  `editVerfifiedTasksEnabled` or `isEditVerfifiedTasksEnabled` and `standaloneModeEnabled` or `isStandaloneModeEnabled` ?